erpnext/patches/v13_0/set_per_picked_in_sales_order.py

```diff
@@ -0,0 +1,14 @@
+import frappe
+
+def execute():
+	frappe.reload_doc('selling', 'doctype', 'sales_order', force=True)
+	sales_orders = frappe.get_all("Sales Order")
+
+	for order in sales_orders:
+		pick_list_items = frappe.get_all("Pick List Item", filters={"sales_order":order.name}, fields= ["sum(qty) as total_qty", "sum(picked_qty) as picked_qty"], group_by="item_code")
```

Comment:
Only get submitted pick lists:

```suggestion
		pick_list_items = frappe.get_all("Pick List Item",
			filters={"docstatus": 1, "sales_order": order.name},
			fields= ["sum(qty) as total_qty", "sum(picked_qty) as picked_qty"],
			group_by="item_code")
```